### PR TITLE
change how socialCapital is updated

### PR DESCRIPTION
**Pull request by EdvardJSHolen**

### CODE REVIEW

File: pkg/infra/teams/team1/q_values.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+func cooperationQ(state state.AgentState) [3]float64 {
+	// TODO: Implement with q-function
+	return [3]float64{0.0, 0.0, 0.0}
+}
+
+func selfishQ(state state.AgentState) [3]float64 {
+	// TODO: Implement with q-function
+	return [3]float64{0.0, 0.0, 0.0}
+}
```
Comment (sherwin-dc) [inline pkg/infra/teams/team1/q_values.go:25]: What do each of the three values represent?

File: pkg/infra/teams/team1/social_capital.go [hunk 2]
```diff
@@ -6,20 +6,6 @@ import (
...
-		return [4]float64{0.0, 0.0, 0.1, 0.1}
-	case decision.Defend:
-		return [4]float64{0.0, 0.0, 0.1, 0.1}
-	default:
-		return [4]float64{0.0, 0.0, 0.0, 0.0}
-	}
-}
-
 // Ensures a float is between -1 and 1
 func boundFloat(inputNumber float64) float64 {
```
Comment (sherwin-dc) [inline pkg/infra/teams/team1/social_capital.go:10]: I think a lot of functions in this file can be moved to `helper_functions`

Comment (EdvardJSHolen) [reply pkg/infra/teams/team1/social_capital.go:10]: Agree, will do that now.

**Review by Razvan00Rusu — APPROVED** [review]
lgtm

**Pull request merged**